### DISCUSSION TITLE
today is not my day ;-( turtle parsing "error"

### DISCUSSION THREAD

**Discussion by naturzukunft** [eclipse-rdf4j/rdf4j#3469] [Q&A] [answered]
sorry for the stupid question, but unfortunately I do not see the problem
(rdf4j 3.7.2 and 3.7.4)

```	
       public static void main(String[] args) throws RDFParseException, UnsupportedRDFormatException, IOException {
		String content = "@prefix haystack: <http://haystack.csail.mit.edu/> \n"
		+ "@prefix person: <http://example.org/person#> \n"
		+ "haystack:ted   a                someprefix:student . \n"
		+ "haystack:mihir a                someprefix:student . \n";
		
		System.out.println(content);
		Rio.parse(new StringReader(content), RDFFormat.TURTLE);
	}
```

->

`Exception in thread "main" org.eclipse.rdf4j.rio.UnsupportedRDFormatException: Did not recognise RDF format object Turtle (mimeTypes=text/turtle, application/x-turtle; ext=ttl)`

save my day
Fredy

**Comment by barthanssens** [comment, marked as answer]
Hi,

hmz, did you include the rdf4j-rio-turtle dependency in your project's pom.xml ?

Best regards

Bart

**Reply by naturzukunft** [reply]
Thank you very much! I absolutely missunderstood the error message.
